Tesis Doctorals en Xarxa.js

```diff
@@ -0,0 +1,309 @@
+
+{
+	"translatorID": "dd2d7c63-0c4b-4da4-b1d8-e72c7445c1e0",
+	"label": "Tesis Doctorals en Xarxa",
+	"creator": "CSUC",
+	"target": "^https?://(www\\.)?(tdx\\.cat|tesisenred\\.net)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-09-22 15:33:33"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+    
+    Tesis Doctorals en Xarxa Translator
+    Copyright © 2017 CSUC
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	var type = ZU.xpath(doc, '//meta[@name="DC.type"]/@content');
+	if (type.length>0) {
+		if (mappingTable[type[0].textContent]) {
+			return mappingTable[type[0].textContent];
+		} else {
+			return "thesis";
+		}
+	}else{
+		return "multiple";
+	}
+}
+
+var mappingTable = {
+	"info:eu-repo/semantics/doctoralThesis":"thesis",
+}
+
+var mappingRights = {
+	"info:eu-repo/semantics/embargoedAccess":"Embargoed Access",
+	"info:eu-repo/semantics/openAccess":"Open Access",
+}
+
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		
+		var hits=[];
+		var urls=[];
+		var results=ZU.xpath(doc,'//div[@class="artifact-title"]/a');
+		for (var i in results) {
+			hits.push(results[i].href);
+		}
+		
+		Zotero.selectItems(hits, function (items) {
+			if (!items) {
+				return true;
+			}
+			var items = new Array();
+			for (var j in items) {
+				urls.push(j);
+				
+			}
+			ZU.processDocuments(urls, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+			
+	var translator = Zotero.loadTranslator('web');
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');//https://github.com/zotero/translators/blob/master/Embedded%20Metadata.js
+	translator.setDocument(doc);
+	translator.setHandler('itemDone', function (obj, item) {
+			//title
+			var titles = ZU.xpath(doc, '//meta[@name="citation_title"]');
```

Comment:
I am quite sure that all these metadata fields are already capture, when you arise her. Can you comment out everything in this function expect the `item.complete();`? What is the result then and which parts are missing then? I would expect that only very few lines will be needed in the end here in this `setHandler` function.